GUI/TaskWindows/SearchTaskWindow.xaml

```diff
@@ -226,6 +227,18 @@
                                         </TextBox.ToolTip>
                                     </TextBox>
                                 </StackPanel>
+                                <StackPanel Orientation="Horizontal" Margin="5 5 5 0" Grid.Column="0" Grid.Row="5">
+                                    <Label Content="Max Threads:" Width="146" />
```

Comment:
Could you change the width to 150 to match others in column? It's a tiny bit out of alignment.
![image](https://user-images.githubusercontent.com/16342951/159804811-d0a07d20-9399-4122-9241-b019ba1c44c1.png)